azure-quantum/tests/unit/test_qiskit.py

```diff
@@ -659,3 +680,36 @@ def test_qiskit_get_qci_qpu_targets(self):
         assert "qir.v1" == config.azure["input_data_format"]
         assert "microsoft.quantum-results.v1" == config.azure["output_data_format"]
 
+    # @pytest.mark.parametrize("endian_pos, expectation",
+    #     [(0,"000 000 001"), (1,"000 010 000"), (2,"100 000 000")]
+    # )
+    @pytest.mark.qci
+    @pytest.mark.live_test
+    def test_qiskit_endianness_submit_to_qci(self, endian_pos=0, expectation="000 000 001"):
+        with unittest.mock.patch.object(
+            Job,
+            self.mock_create_job_id_name,
+            return_value=self.get_test_job_id(),
+        ):
+            workspace = self.create_workspace()
+            provider = AzureQuantumProvider(workspace=workspace)
+            backend = provider.get_backend("qci.simulator")
+            shots = 100
+
+            circuit = self._endianness(pos=endian_pos)
+            circuit.metadata = { "some": "data" }
+
+            qiskit_job = backend.run(
+                circuit=circuit,
+                shots=shots
+            )
+            assert qiskit_job._azure_job.details.metadata["num_qubits"] == '3'
+
+            # Make sure the job is completed before fetching the results
+            self._qiskit_wait_to_complete(qiskit_job, provider)
+
+            assert JobStatus.DONE == qiskit_job.status()
+            result = qiskit_job.result()
+            print(result)
+            assert sum(result.data()["counts"].values()) == shots
+            assert result.data()["counts"][expectation] == shots
```

Comment:
```suggestion

            if JobStatus.DONE == qiskit_job.status():
                result = qiskit_job.result()
                print(result)
                assert sum(result.data()["counts"].values()) == shots
                assert result.data()["counts"][expectation] == shots
```
Instead of `assert`, could you please just check status and check results only if job succeeds.
When we run these tests live there might be some service problems, so we just report warnings if the job fails/timeouts. We only check results if the job actually succeeds